### PR TITLE
RW-C-1: Be more explicit about what these "exact values" are.

### PR DESCRIPTION
**Pull request by cabo**
COMMENT 1 from Rob Wilton.

### CODE REVIEW

**Review by glyn — APPROVED** [review]
LGTM thanks.

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -538,8 +538,9 @@ requirements posed by this document, which are:
 
 1. Integer numbers in the JSONPath query that are relevant
 to the JSONPath processing (e.g., index values and steps) MUST be
-within the range of exact values defined in I-JSON {{-i-json}}, namely
-within the interval \[-(2<sup>53</sup>)+1, (2<sup>53</sup>)-1].
+within the range of exact integer values defined in I-JSON (see {{Section
+2.2 of -i-json}}), namely within the interval \[−(2<sup>53</sup>)+1,
+(2<sup>53</sup>)−1].
```
Comment (gregsdennis) [inline draft-ietf-jsonpath-base.md:543]: Is "integer" really necessary here?  "Integer numbers" is already the context from line 539.

Comment (cabo) [reply draft-ietf-jsonpath-base.md:543]: It is not necessary (that's why it wasn't in there), but we have at least one data point that a reader would consider it helpful.  Do you see a downside?

Comment (gregsdennis) [reply draft-ietf-jsonpath-base.md:543]: Not really. Just felt redundant.

**Review by gregsdennis — APPROVED** [review]

**Pull request merged**